### DISCUSSION TITLE
After Upgrade of JCE, [[[SITE_TOOLBAR]]] is not translated anymore

### DISCUSSION THREAD

**Discussion by linpi1** [joomengine/Joomla-Component-Builder#947] [Q&A] [answered]
I wrote a component last year with JCE version 2.12.16 and decided know to upgrade JCE, step by step the last version 3.1.12. 
In a site view with a custom button, the placeholder [[[SITE_TOOLBAR]]] is not translated any more and appears as [[[SITE_TOOLBAR]]]
In the view.html.php, the button code in the toolbar is there as expected. 
Does some one has any idea how o fix this ?

**Comment by Llewellynvdm** [comment, marked as answer]
All bug related issues please use https://git.vdm.dev/joomla/Component-Builder/issues as I will not respond to bugs, or possible bugs over here, thanks!
